rpcs3/Emu/RSX/VK/VKDMA.cpp

```diff
@@ -279,7 +279,14 @@ namespace vk
 		}
 #else
```

Comment:
Ups, I missed this. Actually you can transform this into an #else if defined(__FreeBSD__)